src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -38,25 +38,18 @@ public void SendAsync_SimpleGet_Success()
         }
 
         [Fact]
-        public async Task GetAsync_Cancel_CancellationTokenPropagates()
```

Comment:
Agreed. Please add the test to the HttpClient.Functional suite within this PR to make sure we don't forget about it.